knowledge-base/chart-area-gradient-color.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Apply gradient effect on Area Chart
+description: How to apply gradient as color for Area Chart series?
+type: how-to
+page_title: Apply gradient effect on Area Chart
+slug: chart-area-gradient-color
+position: 
+tags: chart, area, gradient, color
+ticketid: 1591373
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Chart for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+How can I apply a gradient color effect on the Telerik UI for Blazor Area Chart?
+
+## Solution
+
+1. Declare an external gradient.
```

Comment:
This step is not actionable and does not bring value. Provide a few more words how to define the gradient. There is no problem to link official third-party documentation for general knowledge.

```suggestion
1. Declare an external [linear gradient with an `svg` tag](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/linearGradient).
```